spec/integration/default_spec.rb

```diff
@@ -206,7 +178,7 @@
 end
 
 describe command('gem list') do
-  its(:stdout) { should include('rake (11.1.0)') }
+  its(:stdout) { should match(/^rake \(.*11.1.0.*\)/) }
```

Comment:
Another rake version is installed in the docker image that is used for `itamae local` command test, so relax the test.